README.md

```diff
@@ -254,9 +73,3 @@ If you don't have time, but you need this project to work, or resolve an existin
 
 - Mateusz "[xeho91](https://github.com/xeho91)" Kadlubowski
 - Taku "[ciscorn](https://github.com/ciscorn)" Fukada
-
-## License
-
-![Project License](https://img.shields.io/github/license/svelte-docgen/svelte-docgen?style=for-the-badge)
-
-This project is licensed under the [MIT License](./LICENSE.md).
```

Comment:
GitHub already has a way to show the license.
![image](https://github.com/user-attachments/assets/2b46392f-0c45-4e55-9c85-d0d59c73069e)

And I was aiming to make it more minimal, tidy.